### DISCUSSION TITLE
Which folder do I upload media to via SMB?

### DISCUSSION THREAD

**Discussion by SRoy89** [photoprism/photoprism#2353] [Q&A] [answered]
I'm trying to upload images via smb via photosync app, but I have no idea which folder to put them in.
I would just put all the files in one folder, but im not sure why Photoprism makes folders `02` in `2021` but `01` & `03` are in `2022` folder.

![image](https://user-images.githubusercontent.com/61577078/170039309-19a15b1a-1777-46e5-8ed0-ff02b671c96d.png)

(I'm on android so photoprism option isn't availabe on photosync app, [source](https://www.reddit.com/r/photoprism/comments/prfvri/comment/hdkarav/?utm_source=share&utm_medium=web2x&context=3))

Putting images in `/photoprism/Import`  folder doesn't work because it doesn't show up even after manual indexing. During normal upload via the web ui, photoprism automatically makes a folder with a gibberish name in `import` folder, indexes it, moves it to `2022`, then deletes the gibberish folder.

Please help. Thanks.

**Comment by graciousgrey** [comment, marked as answer]
PhotoPrism offers two ways of adding files: import or index.

 --> https://docs.photoprism.app/user-guide/library/

Importing will sort photos into year/month folders. Indexing keeps your folder structure.

If you decide yo use import you can use the web upload or let PhotoSync sync files to your import folder.

If you decide to use index let PhotoSync sync files to your originals folder.

Here's a detailed guide how to configure PhotoSync

https://docs.photoprism.app/user-guide/sync/mobile-devices/

As you are on android you need to choose webdav as target instead of PhotoPrism.
Make sure to use `/import/` or `/originals/` as directory rather than `Import` or `Originals`

**Reply by SRoy89** [reply]
I see, Ig I shouldn't have tried to use SMB, thanks!

I got WebDav to work. So I uploaded an image to `/import/`, which got indexed after a few mins and it was renamed and copied to the `/originals/`(Pictures) folder.

**But** the original image was not moved or deleted from `/import/`. I hope it's not intentional and I'm doing something wrong, as making copies of the same media is unnecessary and will waste a lot of space.

If there isn't a fix for this, I guess I'll have to upload directly to `/originals/`

![image](https://user-images.githubusercontent.com/61577078/170057157-4709d69e-2f1f-42f2-af37-9f1187a5ba98.png)
![image](https://user-images.githubusercontent.com/61577078/170057275-0ef1e12b-b8c7-4fe3-8900-1a14e29166e0.png)

**Reply by lastzero** [reply]
Change the setting in Library > Import
